spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/catalog/KubernetesCatalogWatchAutoConfiguration.java

```diff
@@ -41,12 +41,12 @@
 @ConditionalOnDiscoveryEnabled
 @ConditionalOnCloudPlatform(CloudPlatform.KUBERNETES)
 @AutoConfigureAfter({ KubernetesClientAutoConfiguration.class })
-public class KubernetesCatalogWatchAutoConfiguration {
```

Comment:
this is a configuration class and imho, messing with visibility accesses is safe for them.